### ISSUE TITLE
Build failure

### ISSUE THREAD

**Issue by israel-hdez** [kiali/kiali#2508]
Starting March 3rd, something went in and started causing a build failure. This is the reason of the missing 1.15.0-snapshot.1 release (no snapshot was released March 6th).

The log of a failed build is attached:

[build-log.txt](https://github.com/kiali/kiali/files/4320026/build-log.txt)

Looks like a mock is missing on GetToken function.

**Comment by israel-hdez** [comment]
Adding the blocker label, because this needs to be fixed so that the release is generated correctly.

**Comment by lucasponce** [comment]
I think is related the config of the CI
I'm goinjg to take a look to see if I can help with the build-log.txt and reproduce locally.

**Comment by lucasponce** [comment]
The command used by Jenkins CI:
```
 make -f deploy/jenkins-ci/Makefile -C . backend-test
```

seems to work locally.

So, still looking at the build-log.txt.

**Comment by lucasponce** [comment]
Ah, I think I got something:

```
GOROOT=/home/lponce/Software/go #gosetup
GOPATH=/home/lponce/Projects/gopath #gosetup
/home/lponce/Software/go/bin/go test -c -o /tmp/___TestAppGraph_in_github_com_kiali_kiali_graph_api github.com/kiali/kiali/graph/api #gosetup
/home/lponce/Software/go/bin/go tool test2json -t /tmp/___TestAppGraph_in_github_com_kiali_kiali_graph_api -test.v -test.run ^TestAppGraph$ #gosetup
=== RUN   TestAppGraph
!!! Set up standard mock
E0313 12:01:04.944846   87522 layer.go:40] Error initializing Kiali Cache. Details: unable to load in-cluster configuration, KUBERNETES_SERVICE_HOST and KUBERNETES_SERVICE_PORT must be defined
--- PASS: TestAppGraph (0.32s)
PASS
```

Some tests seem to swallow some error.
I'll try to address this.

**Comment by lucasponce** [comment]
Fixed by #2515
